apis/marathon.py

```diff
@@ -44,50 +44,33 @@ def create_run():
         return jsonify(id)
 
     return "Error submitting prompt"
-
-"""
-@marathon_api.post('/gen/')
-@check_admin
-def create_marathon_prompt():
-    
-    print("Received marathon prompt req")
-    print(request.json)
     
+@marathon_api.patch('/update_anonymous')
```

Comment:
Use the `@check_request_json` decorator in [`util/decorators.py`](util/decorators.py) to check the request json here.